src/lib/traitify.js

```diff
@@ -1,6 +1,14 @@
-import Traitify from "traitify-client";
-Traitify.ui = require("traitify-ui").default;
-Traitify.ui.client = Traitify;
-Traitify.__version__ = __VERSION__;
+
+/*global __VERSION__ */
+import TraitifyLib from "traitify-client";
 require("polyfills");
-export default Traitify;
\ No newline at end of file
+import ui from "traitify-ui";
+function Init(){
```

Comment:
The reason for requiring the init is that you then can force webpack to make a new copy of the required return every time its required

i.e. if you just return an object webpack will share required modules, but if you put it in a function and then call that function and within that scope set Traitify to extend TraitifyLib then you get a new version of the TraitifyLib every time